packages/slice-machine/yarn.lock

```diff
@@ -1600,7 +1600,7 @@
     "@nodelib/fs.scandir" "2.1.5"
```

Comment:
Why is there so many changes in the yarn lock ?

Reply:
I used lerna to reinstall all the dependancies on all the packages :)